### ISSUE TITLE
Plugin appears to be called, but no rendering happens?

### ISSUE THREAD

**Issue by vputz** [lydian/reviewboard-ipynb#1]
Trying very much to find a notebook review tool for us, and this seems very close to working, but not quite yet!

Using ikatson's reviewboard-docker (reviewboard 3.0.14, about a year ago) I can install the plugin by modifying the container to "pip install reviewboard-ipynb"; it then appears in the list of extensions in the admin interface, and if I add a notebook file I can see in the logs

```
GET /static/ext/reviewboard_ipynb.extension.IpynbReviewUIExtension/css/default.min.e1f8599dff94.css => generated 0 bytes in 0 msecs (HTTP/1.1 304) 0 headers in 29 bytes (0 switches on core 0)
```

So it does seem to be installed.  But the attached notebook document is completely blank and cannot be viewed for comment (see attached).

I am not certain if this is user error or an actual problem, but am unsure how to diagnose or fix the issue.  Right now our best option seems to be exporting to markdown (which misses a lot of the notebook).  Any suggestions or other information I can get you?

![image](https://user-images.githubusercontent.com/4076914/92157928-c6b66e80-edf0-11ea-97fc-2a2ee7d1bdcf.png)

**Comment by lydian** [comment]
hmm, usually it is due to the mime type mismatch.  Is it possible if you could share the notebook you're testing? I can try to debug for you :)

**Comment by vputz** [comment]
Certainly, and thanks for the help!  It seems to happen with just about any
notebook, so I've duplicated the effect with this basic "hello world"
notebook; same symptoms (blank icon/page in reviewboard with only "comment"
available, no viewing), and at least some mention of the plugin in the
container logs:

2020-09-08T19:36:03.997004851Z [pid: 70|app: -1|req: -1/407] 100.96.3.17 ()
{58 vars in 1269 bytes} [Tue Sep  8 19:36:03 2020] GET
/static/ext/reviewboard_ipynb.extension.IpynbReviewUIExtension/css/default.min.e1f8599dff94.css
=> generated 0 bytes in 1 msecs (HTTP/1.1 304) 0 headers in 29 bytes (0
switches on core 0)

If it's a mime type mismatch, I see in this file "mimetype":
"text/x-python", but that's listed in the supported mimetypes, so it's a
little mystifying to me.

This is with docker container image ikatson/reviewboard:3.0.14 and the
plugin installed with "pip install reviewboard-ipynb" prior to starting the
container's start.sh script (which starts up reviewboard).

I'm glad to provide more information and do some hacking on this side as
needed--apologies for pestering you about it but I'd love to at least give
it a go!  Not sure how to get more diagnostic information...

**Comment by lydian** [comment]
I found the issue is due to I only test this plugin with reviewboard 2.5.16, which is even earlier version. I'll work on making this compatible with newer version :)
